web/src/hooks/__tests__/useUserLocation.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, waitFor } from '@testing-library/react'
+import React from 'react'
+
+import useUserLocation from '../useUserLocation'
+
+jest.mock('../useUserLocation', () => ({
+  __esModule: true,
+  default: jest.fn(),
+  getUserLocation: jest.fn(),
+}))
```

Comment:
:wrench: Hmm... you are basically testing that the mock returns exactly what you mock it to return. The only thing that should be mocked in this test file is `navigator.geolocation.getCurrentPosition`. Can you try to adjust your tests to actually test something? If you need help, let me know :)